testData/solutions/win32.json

```diff
@@ -940,6 +940,38 @@
     },
 
     {
+        "name": "Windows Display Settings",
+        "id": "com.microsoft.windows.displaySettings",
```

Comment:
Entries in solutions registry should now be keyed by solutionID and the 'id' inside the solutions registry entry is no longer necessary